students/timm67/session05/mailroom_test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env/python3
+
+"""
+Tim Meese
+Au2018-Py210B
+Mailroom Part 4 assignment
+"""
+
+import pytest
+from mailroom_part4 import create_report_task
+from mailroom_part4 import send_thankyou_multiple_donors_task
+from mailroom_part4 import send_thankyou_single_donor_test
+from mailroom_part4 import add_single_donor_test
+from pathlib import Path
+
+# Duplicate donor database
+
+donors = {
+     ('Jim', 'Tillson'): [5.00, 20.00],
+     ('Barb', 'Langley'): [10.00, 20.00, 100.00],
+     ('Jen', 'Garfield'): [10.00, 20.00, 5.00],
+     ('Rex', 'Miller'): [20.00, 20.00, 5.00],
+     ('Tony', 'Blake'): [20.00, 20.00, 10.00]
+     }
+
+thankyou_file_list = []
+
+
+def test_report():
+    outlines = create_report_task()
+    assert(5 == len(outlines))
```

Comment:
No need to change, but it would have been better to check the content of the `outlines` list as well, not just the number of items.